Build failure: papis

### Steps To Reproduce

Steps to reproduce the behavior:
1. build papis

### Build log

```
$ nix shell nixpkgs#papis        
error: builder for '/nix/store/jah5bvsyrqnhamaw2vsbmrll67h2r51x-python3.11-papis-0.13.drv' failed with exit code 1;
       last 10 log lines:
       >
       > tests/database/test_whoosh.py::Test::test_add
       >   /nix/store/apmlv3n5m2hayb1mmmq02iyxhzbzwk1m-python3.11-whoosh-2.7.4/lib/python3.11/site-packages/whoosh/codec/whoosh3.py:1116: SyntaxWarning: "is" with a literal. Did you mean "=="?
       >     elif fixedsize is 0:
       >
       > -- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
       > =========================== short test summary info ============================
       > FAILED tests/test_config.py::test_get_configuration - assert 140737327197392 == 140737259635920
       > ==== 1 failed, 212 passed, 1 skipped, 30 deselected, 11 warnings in 24.85s =====
       > /nix/store/c8dj731bkcdzhgrpawhc8qvdgls4xfjv-stdenv-linux/setup: line 1578: pop_var_context: head of shell_variables not a function context
       For full logs, run 'nix log /nix/store/jah5bvsyrqnhamaw2vsbmrll67h2r51x-python3.11-papis-0.13.drv'.
```

### Notify maintainers

@nico202 @teto @marsam

### Metadata

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
> nix-shell -p nix-info --run "nix-info -m" 
 - system: `"x86_64-linux"`
 - host os: `Linux 6.6.21, NixOS, 24.05 (Uakari), 24.05.20240321.20f77aa`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.2`
 - nixpkgs: `/nix/store/wa8lhchql76i6czzvad3bqz6lvnjnbr5-source`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/python-modules/papis/default.nix
+++ b/pkgs/development/python-modules/papis/default.nix
@@ -99,6 +99,7 @@ buildPythonPackage rec {
     "test_downloader_getter"
     "test_general"
     "test_get_config_dirs"
+    "test_get_configuration"
     "test_get_data"
     "test_valid_dblp_key"
     "test_validate_arxivid"

